tests/sentry/sentry_metrics/test_gen_metrics_multiprocess_steps.py

```diff
@@ -424,7 +424,7 @@ def test_process_messages_default_card_rollout(set_sentry_option: Callable[...,
     ),
     (
         {
-            "name": "c:transactions/alert@none" * 21,
+            "name": "c:transactions/alert@" + ("none" * 50),
```

Comment:
nit: this looks unrelated

Reply:
Oh yeah, it messed up since I rebased the PR. Will figure out why.